### PR TITLE
address findbug issues

### PR DESCRIPTION
**Pull request by zliu41**
Addressed two remaining findbugs warnings for gobblin-compaction.

### CODE REVIEW

File: gobblin-compaction/src/main/java/gobblin/compaction/CompactionRunner.java [hunk 1]
```diff
@@ -51,8 +51,8 @@
   private static final String DATAFORMAT_EXTENSION_NAME = ".dataformat.extension.name";
   private static final String OUTPUT = "output";
 
-  public static Properties properties = new Properties();
-  public static Properties jobProperties = new Properties();
+  static Properties properties = new Properties();
```
Comment (liyinan926) [inline gobblin-compaction/src/main/java/gobblin/compaction/CompactionRunner.java]: It seems properties is only used in main(), so you can make it a local variable inside main(). And jobProperties can be private static.

Comment (zliu41) [inline gobblin-compaction/src/main/java/gobblin/compaction/CompactionRunner.java]: They are both used in multiple other classes, e.g., HiveJdbcConnector, HdfsIO, SerialCompactor

Comment (liyinan926) [inline gobblin-compaction/src/main/java/gobblin/compaction/CompactionRunner.java]: I see.

**Comment by liyinan926** [comment]
LGTM.

**Pull request merged**